### PR TITLE
MTA-557 Docs Improvement - Automated tagging of resources

### PR DESCRIPTION
**Pull request by sbeskin-redhat**
Resolves: https://issues.redhat.com/browse/MTA-557
MTA 6.1.0+
Preview: 
https://deploy-preview-705--windup-documentation.netlify.app/docs/web-console-guide/master/index.html#mta-web-tagging-an-application_user-interface-guide

### CODE REVIEW

**Comment by sbeskin-redhat** [comment]
@rromannissen 
Please review this PR.

**Comment by rromannissen** [comment]
@sbeskin-redhat looks good to me!

File: docs/topics/mta-web-tagging-an-application.adoc [hunk 1]
```diff
@@ -26,6 +28,8 @@ The *Update application* window opens.
 [id="automating-tagging-of-an-application_{context}"]
 == Automatic tagging
 
+{ProductShortName} translates the technology stack information that the analysis module collects into tags and automatically adds them to the application. Automatic tagging is especially useful when dealing with large portfolios of applications.
+
```
Comment (RichardHoch) [inline docs/topics/mta-web-tagging-an-application.adoc:32]: Suggestion "... and automatically adds the tags to the application."

**Comment by RichardHoch** [comment]
One small suggestion; otherwise, LGTM. Can be merged with or without applying the suggestion.

**Pull request merged**